packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -1457,3 +1457,9 @@ bool isPrimativeInstanceKind(String? kind) {
       kind == InstanceKind.kNull ||
       kind == InstanceKind.kString;
 }
+
+// TODO(mtaylee): Prefer to use this helper method whenever a call to
+// .split('/').last is made on a String (usually on URIs).
+/// Returns the file name from a URI or path string, by splitting the [uri] at
+/// the directory separators '/', and returning the last element.
+String? fileNameFromUri(String? uri) => uri?.split('/').last;
```

Comment:
@bkonyi will this work for a windows uri? Or does the VM send package uris always with `/` and never with `\` (event for a windows desktop app).

Reply:
I'm 99% sure it will. I don't think the VM itself knows how to handle `\`'s internally, only in `dart:io`, so URIs from the service should be using forward slashes.